<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>252</width>
     <height>157</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="boton_cargar_mat">
      <property name="styleSheet">
       <string notr="true">QPushButton#boton_cargar_mat{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20,47, 78, 219), stop:1 rgba(85, 98, 112, 255));
	color:rgba(255,255,255,210);
	border-radius:5px
}
QPushButton#boton_cargar_mat:hover{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(40,67, 98, 219), stop:1 rgba(105, 118, 132, 255));
	color:rgba(255,255,255,210);
	border-radius:5px
}
QPushButton#boton_cargar_mat:pressed{
	padding-left:5px;
	padding-top:5px;
background-color:rgba(105,118,132,200);
}</string>
      </property>
      <property name="text">
       <string>Cargar archivo .MAT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>            SELECCIONE LA LLAVE:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item>
     <widget class="QPushButton" name="boton_CSV">
      <property name="styleSheet">
       <string notr="true">QPushButton#boton_CSV
{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20,47, 78, 219), stop:1 rgba(85, 98, 112, 255));
	color:rgba(255,255,255,210);
	border-radius:5px
}
QPushButton#boton_CSV:hover{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(40,67, 98, 219), stop:1 rgba(105, 118, 132, 255));
	color:rgba(255,255,255,210);
	border-radius:5px
}
QPushButton#boton_CSV:pressed{
	padding-left:5px;
	padding-top:5px;
background-color:rgba(105,118,132,200);
}</string>
      </property>
      <property name="text">
       <string>Graficar se√±al</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="boton_CSV_2">
      <property name="styleSheet">
       <string notr="true">QPushButton#boton_CSV_2
{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20,47, 78, 219), stop:1 rgba(85, 98, 112, 255));
	color:rgba(255,255,255,210);
	border-radius:5px
}
QPushButton#boton_CSV_2:hover{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(40,67, 98, 219), stop:1 rgba(105, 118, 132, 255));
	color:rgba(255,255,255,210);
	border-radius:5px
}
QPushButton#boton_CSV_2:pressed{
	padding-left:5px;
	padding-top:5px;
background-color:rgba(105,118,132,200);
}</string>
      </property>
      <property name="text">
       <string>Graficar promedio</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="boton_salir">
      <property name="styleSheet">
       <string notr="true">
QPushButton#boton_salir{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20,47, 78, 219), stop:1 rgba(85, 98, 112, 255));
	color:rgba(255,255,255,210);
	border-radius:5px
}
QPushButton#boton_salir:hover{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(40,67, 98, 219), stop:1 rgba(105, 118, 132, 255));
	color:rgba(255,255,255,210);
	border-radius:5px
}
QPushButton#boton_salir:pressed{
	padding-left:5px;
	padding-top:5px;
background-color:rgba(105,118,132,200);
}</string>
      </property>
      <property name="text">
       <string>Salir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-130</x>
     <y>-120</y>
     <width>1141</width>
     <height>851</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,100);
border-radius:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
